internal/pkg/agent/application/monitoring/v1_monitor.go

```diff
@@ -517,6 +518,8 @@ func (b *BeatsMonitor) monitoringNamespace() string {
 }
 
 func (b *BeatsMonitor) injectMetricsInput(cfg map[string]interface{}, componentIDToBinary map[string]string, monitoringOutputName string, componentList []component.Component) error {
+	metricsCollectionInterval := 60 * time.Second
```

Comment:
Any reason not to make this a constant?

Reply:
Also would be great to add a "why" comment for future us.

Reply:
Added a constant...
Not sure what you mean with a "why" comment? You mean "Why do we set it to 1 minute?" type of comment ?

Reply:
Yes, sorry, I should've been clearer.  I think it would be good to know why we're using 1 minute, because typically most of our polling periods (thinking of Metricbeat modules) are 10s by default.

Reply:
The purpose here was to reduce the load/data volume on the monitoring cluster without loosing the ability to get constant data from agent, that's why we ended up choosing 1 minute.

Reply:
> The purpose here was to reduce the load/data volume on the monitoring cluster without loosing the ability to get constant data from agent, that's why we ended up choosing 1 minute.

Right, I got that from the PR description as well.  Let's put it in a comment above the const declaration so someone looking at the code a few months from now easily understands it as well (without needing to find the commit that introduced it -> this PR).

Reply:
done in 4d71ff6642e2586048e98e44921c42a900bdf9d9